seatunnel-connectors-v2/connector-doris/src/main/java/org/apache/seatunnel/connectors/doris/util/DorisCatalogUtil.java

```diff
@@ -153,6 +162,17 @@ public static String getCreateTableStatement(
                         rowTypeFields);
     }
 
+    private static boolean canHandledByDefaultTemplate(
+            String createTemplate, String primaryKey, String uniqueKey) {
+        return (StringUtils.equals(
+                                createTemplate,
+                                DorisOptions.SAVE_MODE_CREATE_TEMPLATE.defaultValue())
+                        && StringUtils.isBlank(primaryKey)
+                        && StringUtils.isBlank(uniqueKey))
```

Comment:
The default template seem like doesn't care uniqueKey value.
```suggestion
                        && StringUtils.isBlank(primaryKey))
```